src/Http/Authentication.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,4 @@
 #nullable enable
+Microsoft.AspNetCore.Authentication.AuthenticationException
+Microsoft.AspNetCore.Authentication.AuthenticationException.AuthenticationException(string? message) -> void
+Microsoft.AspNetCore.Authentication.AuthenticationException.AuthenticationException(string? message, System.Exception? innerException) -> void
```

Comment:
Can you fill out the [API proposal issue template](https://github.com/dotnet/aspnetcore/issues/new?assignees=&labels=api-suggestion&template=30_api_proposal.md&title=) for this? If you prefer, you can copy the template into a new comment on #47708 and fill it out. Or just file a new issue. I'll add the right labels so we go over it in API review Thursday.

Reply:
Open to feedback: https://github.com/dotnet/aspnetcore/issues/47708#issuecomment-1522276638